packages/preview/dndcampaign/0.1.0/typst.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "dndcampaign"
+version = "0.1.0"
+entrypoint = "campaign.typ"
+authors = ["Yan Xin <@yanwenywan>"]
+description = "Easily write DnD 5e style campaign documents."
+categories = ["layout", "text", "book"]
+license = "Apache-2.0"
+keywords = ["rpg", "5e", "dnd", "campaign"]
+repository = "https://github.com/yanwenywan/typst-packages/tree/master/dndcampaign"
+exclude = ["**/*.pdf", "template/fonts/*"]
```

Comment:
The template files should not be excluded: if they are they won't be in the package archive and won't be copied in the end. However in that case, because the fonts files are quite large, I think it would just be best to not include them in the template at all, and give people instructions on how to download them and use them with Typst (in your README and/or in a comment of `template/main.typ`)